torch/_prims/nvfuser_executor.py

```diff
@@ -68,6 +84,21 @@ def make_nvfuser_fusion(gm: GraphModule, *nv_args_templates):
                 f"Node {node} with target {node.target} does not support nvfuser"
             )
 
+    graph_input_nodes = list(filter(lambda n: n.op == "placeholder", gm.graph.nodes))
+    call_function_nodes = list(
+        filter(lambda n: n.op == "call_function", gm.graph.nodes)
+    )
+    assert len(graph_input_nodes) == len(
+        nv_args_templates
+    ), "Number of placeholder nodes in the graph must match number of args"
+    assert len(nv_args_templates) > 0, "There must be at least one argument"
+    assert (
+        len(call_function_nodes) > 0
+    ), "Graph must contain at least one call_function node"
+    assert not _any_get_attr_used(
```

Comment:
Any reason that we are explicitly asserting here?

I remember seeing some issues with aot mistakenly converting things to constant tensor, but I don't know if those are always just wrong.

Reply:
Without this assertion the error comes from the `fusion.execute` call and it's less readable.